solutions/swb-reference/integration-tests/tests/isolated/projects/get.test.ts

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+import ClientSession from '../../../support/clientSession';
+import { PaabHelper } from '../../../support/complex/paabHelper';
+import HttpError from '../../../support/utils/HttpError';
+import { checkHttpError } from '../../../support/utils/utilities';
+
+describe('Get Project negative tests', () => {
+  const paabHelper: PaabHelper = new PaabHelper(2);
+  let pa1Session: ClientSession;
+  let rs1Session: ClientSession;
+  let project2Id: string;
+
+  beforeEach(async () => {
+    expect.hasAssertions();
+  });
+
+  beforeAll(async () => {
+    const paabResources = await paabHelper.createResources();
+    pa1Session = paabResources.pa1Session;
+    rs1Session = paabResources.rs1Session;
+    project2Id = paabResources.project2Id;
+  });
+
+  afterAll(async () => {
+    await paabHelper.cleanup();
+  });
+
+  describe('when a ProjectAdmin GETs a project they do not belong to', () => {
+    test('it returns a 403', async () => {
+      try {
+        await pa1Session.resources.projects.project(project2Id).get();
```

Comment:
Let's also throw an [error](https://github.com/aws-solutions/research-service-workbench-on-aws/pull/1125#discussion_r1223821122) after the API call in case it doesn't throw an error.

Reply:
Test fails if it never runs into an expect.